python/lsst/ts/simactuators/commander_utils.py

```diff
@@ -0,0 +1,265 @@
+# This file is part of ts_simactuators.
+#
+# Developed for the LSST Data Management System.
```

Comment:
Rubin Observatory Telescope and Site System.

Reply:
Good suggestion. Done.